_posts/2020-12-22-aws-antipatterns.md

```diff
@@ -0,0 +1,198 @@
+---
+layout: post
+title: AWS Antipatterns
+author: arto
+excerpt: >
+  Cloud platforms offer unprecedented agility, innovation, and speed. Yet I see the same mistakes being repeated over and over again. This blog post is a collection of some things people often get wrong - and some thoughts on how to dodge those mistakes.
+tags:
+ - AWS
+ - Amazon Web Services
+ - Best Practices
+ - Lessons Learned
+ - Antipatterns
+ - Security
+---
+
+![Speed and agility of cloud development can be unsurpassed](/img/aws-antipatterns/aws_rider.png)
+
+Cloud platforms are incredible. For anyone who used to create or operate software running in a traditional on-premises server room, they offer incredible speed in getting ideas to production, and experimentation with new things. However, this speed comes with a cost: Sometimes that cost is money, as AWS bills can sneak up on you. Sometimes the cost may be a bad architecture that gets more expensive as you progress, a badly compromised security that makes you vulnerable for many new attack vectors - or it can be that you are paying for lots of new services just because you can - without getting the benefits you were looking for. 
+
+In this article, I'm going to go through some of the worst mistakes that might happen when you start working on the AWS platform. These mistakes are based on both personal experience, aka my mistakes, and things I've seen, aka other people's mistakes. I will also try to provide some insights into why we need to pay attention here, and what could we do to avoid the negative impact while benefiting from the newfound speed and agility. I've been working pretty much in cloud and serverless for the last five years, before that I had a lot of on-premises experience. I'm most well versed with AWS architecture and best practices, but most of these thoughts will probably apply also to Azure and Google Cloud Platform projects.
```

Comment:
> These mistakes are based on both personal experience, aka my mistakes, and things I've seen, aka other people's mistakes

A bit of repetition on aka :) others' mistakes are implied on the "some of the mistakes are personal", right? How about:
Some of the mistakes here are personal 😓 .

Reply:
I like this repetition, it's intentional here :)